frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx

```diff
@@ -37,6 +37,10 @@ export const CreateOCSServiceForm: React.FC<CreateOCSServiceFormProps> = ({
         <p className="co-legend co-required ceph-ocs-desc__legend">
           Select at least 3 nodes in different failure domains you wish to use.
         </p>
+        <p>
+          3 selected nodes are used for deployment and the remaining nodes that are available are
```

Comment:
The text doesn't look good to me.

```suggestion
          The selected nodes will be used for deployment and the remaining nodes can be used for scaling purposes.
```

Reply:
I think, number `3` is important here as per the conversations on https://bugzilla.redhat.com/show_bug.cgi?id=1778061.